src/hotspot/cpu/x86/stubGenerator_x86_64_poly.cpp

```diff
@@ -559,7 +559,7 @@ void StubGenerator::poly1305_process_blocks_avx512(
   const Register t0 = r13;
   const Register t1 = r14;
   const Register t2 = r15;
-  const Register rscratch = r13;
+  const Register rscratch = r14;
```

Comment:
The register map above in the comments also should reflect this change that rscratch is r14 now.

Reply:
done.  (or rather, ended up removing `rscratch` completely from that function)